### ISSUE TITLE
ignored directories

### ISSUE THREAD

**Issue by SteveRohrlack** [tib/lint.swift#1]
this is a pure enhancement suggestion:

allow configuration of ignored directories as they are currently hard-coded 

optionally respect entries in the project's .gitignore

**Comment by tib** [comment]
yes the configuration can be useful, but i don't feel the need of it. 
usually i just want to omit the pods and carthage directories, nothing more...

if you want you can implement this feature and send me a pull request, i'll merge into the repository. ;)
